### PR TITLE
Milestone 5: Views

### PR DESCRIPTION
**Pull request by tresorsawasawa**
In this PR, I completed the following tasks:
- [ ] Implemented the design from the sneak peek wireframes.
- [ ] Used methods that I have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- [ ] Implemented index and show function in User and Post controller
- [ ] Used partials

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,28 +1,16 @@
...
-        <span class="comments-likes">Comments: x, Likes: x</span>
-      </div>
-      <div class="user-bloc-post-comments">
-        <% for i in 1..5%>
-          <span> Username: Comment <%= i %> </span>
-        <% end %>
-      </div>
-    </div>
+  <% @user.posts.each do |post| %>
+      <%= render partial: "post", locals: { post: post } %>
```
Comment (andersonlebon) [inline app/views/posts/index.html.erb:13]: - [x] KIndly make sure when I click on the post, it should bring me to the `post#show` page.

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1,11 +1,5 @@
...
-  <% for i in 0..9 do %>
-    <div class="user-bloc">
-      <div class="user-image"></div>
-      <div class="user-data">
-        <h4 class="title">User name</h4>
-        <span class="user-posts">Number of posts: x</span>
-      </div>
-    </div>
+  <% @users.each do |user| %>
+    <%= render partial: 'user', locals: {user: user} %>
```
Comment (andersonlebon) [inline app/views/users/index.html.erb:3]: image


- [x] kindly make sure when I click on the user it should bring me to the `user#show` template/view

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @tresorsawasawa 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tresorsawasawa** [comment]
Hey @andersonlebon 

Thank You for the review :pray: 
Thank You
I really appreciate :+1: :+1:

**Review by andersonlebon — APPROVED** [review]
## Hi @tresorsawasawa 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by tresorsawasawa** [comment]
Hey @andersonlebon

Thank You for the review :pray: 
Thank You
I really appreciate :+1: :+1:

**Pull request merged**